### PR TITLE
Add UT for Azure Ops during packaging 

### PR DESCRIPTION
**Pull request by RandySheriffH**
Add unit test cases for AzureEP ops, which are disabled by default.
On Linux/windows packaging, cases would be enabled by a pipeline flag.

### CODE REVIEW

File: .pipelines/OneBranch.PullRequest.yml [hunk 1]
```diff
@@ -16,13 +16,10 @@ parameters: # parameters are shown up in ADO UI in a build queue time
...
-- name: BuildAzureOp
-  displayName: 'Build and test AzureEP ops'
+- name: ExtraEnv
+  displayName: 'Extra env variable set to CIBW_ENVIRONMENT'
   type: string
-  values:
-  - '0'
-  - '1'
-  default: '0'
+  default: 'None=None'
```
Comment (wenbingl) [inline .pipelines/OneBranch.PullRequest.yml:22]: why not just ''?

Comment (RandySheriffH) [reply .pipelines/OneBranch.PullRequest.yml:22]: ci disallow empty string as default.

File: .pipelines/wheels_linux.yml
[the remark this answers is not in the document]
Comment (RandySheriffH) [reply .pipelines/wheels_linux.yml:1]: AzureOps does not support mac.

File: .pipelines/OneBranch.PullRequest.yml [hunk 2]
```diff
@@ -108,6 +105,7 @@ extends:
             python -m pip install --upgrade pip
             python -m pip install cibuildwheel numpy
             python -m cibuildwheel --platform windows --archs AMD64 --output-dir $(REPOROOT)\out
+            tools\install_deps.bat uninstall
```
[the remark this answers is not in the document]
Comment (RandySheriffH) [reply .pipelines/OneBranch.PullRequest.yml:108]: that's right, just added.

**Review by wenbingl — APPROVED** [review]

**Pull request merged**